docs/settingup.md

```diff
@@ -436,9 +436,34 @@ Clear all selections in an app.
 
 ## ClickActionButton action
 
-Simulate clicking an action-button
-
-
+A `ClickActionButton`-action simulates clicking an _action-button_. An _action-button_ is a chart item which, when clicked, executes a series of actions. The series of actions contained by an action-button begins with any number _generic button-actions_ and ends with an optional _navigation button-action_.
```

Comment:
Chart item? Sheet item, perhaps?

Reply:
Fixing, using that term is just confusing. It is a chart in some meaning though 😄 
![image](https://user-images.githubusercontent.com/67060976/90391222-0ceb9e00-e08d-11ea-9059-1ba67ca8703b.png)